src/tests/specs/detailPage.spec.ts

```diff
@@ -110,4 +110,18 @@ describe('Detail View test: ', () => {
     await planner.detailPage.stateToggle.untilTextIsPresent('open');
     expect(planner.detailPage.stateToggle.getTextWhenReady()).toContain('open');
   });
+
+  xit('Should change the type of work item', async () => {
```

Comment:
Why is it under `xit` ?

Reply:
Because since the backend is not ready the test will fail as the type will not get updated